spec/support/acceptance/shared_contexts.rb

```diff
@@ -27,22 +27,26 @@
   end
 end
 
-RSpec.shared_context 'corrective change' do |noop = false|
+# NOTE: The incident query setup also cleans any dangling ServiceNow instances and
+# sets up the master's site.pp for the incident tests. Thus, make sure to place this
+# _before_ the incident query setup so that the latter's postconditions are still
+# maintained (like the master's expected site.pp).
+RSpec.shared_context 'corrective change setup' do |file_resource_hash|
```

Comment:
Using the resource hash makes it easier to setup tests where we'd like a report with multiple corrective changes. I had tests like that in a previous iteration of this PR, but removed those tests b/c I didn't need them. However I still feel that making this a param is still useful.